arangod/Replication2/ReplicatedLog/Algorithms.cpp

```diff
@@ -150,7 +150,7 @@ auto algorithms::updateReplicatedLog(
 
       TRI_ASSERT(spec->participantsConfig.generation > 0);
       auto newLeader = log->becomeLeader(
-          spec->currentTerm->config, myServerId, spec->currentTerm->term,
+          spec->participantsConfig.config, myServerId, spec->currentTerm->term,
```

Comment:
This argument is now superfluous because the participantsConfig is also a parameter for the leader. But we can leave it as is and do the cleanup later.

Reply:
I will do this in a separate PR, because it trickles down quite a bit.